src/BusinessService.ts

```diff
@@ -388,6 +388,12 @@ export type BusinessUpdate = Partial<
   }
 >;
 
+export type BusinessReplace = BusinessUpdate & {
+  verifiedDomains: Array<{
+    domain: string;
+  }>;
+};
+
```

Comment:
Just curious, why is `verifiedDomains` required instead of being optional like in `BusinessUpdate`?

Reply:
Never mind, figured it out 👍